front-end/src/lib/chartOptions.js

```diff
@@ -25,6 +25,16 @@ export const defaultChartOptions = {
   }
 };
 
+export const movePlotLine = (chartOptions, x) => {
```

Comment:
This should be near the bottom of the file, don't need to interrupt the chartOption functionality

Reply:
moved